resolwe_bio/tools/parse_tabular_file.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python2
+"""Parse tabular file."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import argparse
+import csv
+import gzip
+import os
+import xlrd
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Parse tabular file")
+    parser.add_argument('input_file', help="Tabular file.")
+    parser.add_argument('output_file', help="Output file.")
+    return parser.parse_args()
+
+
+def main():
+    """Invoked when run directly as a program."""
+    args = parse_arguments()
+
+    ext = os.path.splitext(args.input_file)[-1].lower()
+    with gzip.open(args.output_file, mode='wt') as outfile:
+        csvwriter = csv.writer(outfile, delimiter=str('\t'), lineterminator='\n')
+
+        try:
+            if ext == '.tab' or ext == 'txt' or ext == 'tsv':
```

Comment:
The dot is missing for txt and tsv. Also, more concise and a general solution is:
```
if ext in ('.tab', '.txt', '.tsv'):
```